update readme to develop locally with another app (#35)
primer/view_components: README.md

```bash
script/storybook
```

### Developing with another app

> In your app, change the Gemfile from

Reviewer: ```suggestion
In your `Gemfile`, change:
```

```ruby
gem "primer_view_components"
```

to

```ruby
gem "primer_view_components", path: "path_to_the_gem" # e.g. path: "~/primer/view_components"
```

Then, `bundle install` to update references. You'll now be able to see changes from the gem without having to build it.

> Remember that restarting Rails server is necessary to see the changes, since the gem is loaded at boot.

Reviewer: ```suggestion
Remember that restarting the Rails server is necessary to see changes, as the gem is loaded at boot time.
```

> To minimize the number of restarts, we recommend checking the component first in Storybook and when it's in a good state

Reviewer: ```suggestion
To minimize the number of restarts, we recommend checking the component in Storybook first, and then when it's in a good state,
